web-client/src/modules/timeline/components/OffersList/OffersList.tsx

```diff
@@ -103,6 +106,7 @@ const ButtonsContainer = styled.div`
 const OfferItem: React.FC<OfferItemProps> = ({
   offer,
   handleOffer,
+  requestCoords,
```

Comment:
I see that in the timeline container, you have passed this as a prop to the offersList and the offersList component isn't passing it down to the offerItem anywhere